doc/site/docs/02-core-concepts/04-requests.md

```diff
@@ -8,13 +8,13 @@ The `Request` object represents an incoming HTTP request to your Relic server. I
 
 Every handler in Relic receives a request through the context object (`ctx.request`), allowing you to inspect and process the incoming data before generating a response.
 
-## Understanding the Request Object
+## Understanding the request object
```

Comment:
Surprised by this lower-case change. Just making sure it is intended.

Reply:
Mistake